cluster/src/main/java/org/apache/iotdb/cluster/log/snapshot/FileSnapshot.java

```diff
@@ -430,24 +427,20 @@ private void loadRemoteResource(RemoteTsFileResource resource) throws IllegalPat
     private File pullRemoteFile(RemoteTsFileResource resource, Node node) throws IOException {
       logger.info("{}: pulling remote file {} from {}, plan index [{}, {}]", name, resource, node
           , resource.getMinPlanIndex(), resource.getMaxPlanIndex());
-
-      String[] pathSegments = FilePathUtils.splitTsFilePath(resource);
-      int segSize = pathSegments.length;
       // the new file is stored at:
-      // remote/<nodeIdentifier>/<storageGroupName>/<partitionNum>/<fileName>
+      // remote/<nodeIdentifier>/resource.getTsFile().getAbsolutePath()
```

Comment:
use logicalStorageGroupName/virtualStorageGroupName/timePartitionId/fileName is more better

Reply:
The path may change, so I put all the annotation in the` splitTsFilePath()` method in FilePathUtils to reduce dispersion so that such bugs occur less often